integrations/extensions/starter-kits/zendesk-support/zendesk.actions.json

```diff
@@ -218,21 +174,22 @@
                 "type": "integration_interaction",
                 "method": "POST",
                 "internal": {
-                  "catalog_item_id": "b166041f-67b2-4c1a-8557-1d9ef3bc30ba"
+                  "spec_hash_id": "a8d14c2e0a330bbd75acc4465f0405f339e75728b5fe2f4e3950d71d926ce132",
+                  "catalog_item_id": "87464cd6-44de-4848-b41c-78d25ab66638"
```

Comment:
This should be `null` .. the `catalog_item_id` is unique per-assistant, setting it to `null` tells store that there will be an error here when the step is imported (which we expect)